docs/guides/src/main/server/containers.adoc

```diff
@@ -71,6 +72,8 @@ INFO  [org.key.com.Profile] (main) Preview feature enabled: token_exchange
 ----
 This message shows the desired feature is enabled.
 
+Health check endpoints are available at `https://localhost:8443/health`, `https://localhost:8443/health/ready` and `https://localhost:8443/live`.
```

Comment:
We should have something that documents these endpoints as right now it's somewhat hidden in an example in the containers guide.

Not sure if it should be an observerability guide, health guide, or just documented in the containers guide. I'd vote for one of the first two, and don't think containers guide is the right place.